nginx/docker-entrypoint.d/20-envsubst-on-templates.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```

Comment:
Cool script, any context on this being something developed for hackerspace.zone or a standard thing with nginx?

Without having tested/dug deep, it seems the nginx docker image supports this kind of envsubst:

https://github.com/docker-library/docs/tree/master/nginx#using-environment-variables-in-nginx-configuration-new-in-119